Libraries/Components/StaticRenderer.js

```diff
@@ -12,18 +12,20 @@
 
 const React = require('React');
 
-const PropTypes = require('prop-types');
-
-class StaticRenderer extends React.Component<{
+type StaticRendererProps = $ReadOnly<{|
+  /**
+   * Indicates whether the render function needs to be called again
+   */
   shouldUpdate: boolean,
+  /**
+   * (props) => renderable
+   * A function that takes in certain props that are used to render a component
+   */
   render: Function,
```

Comment:
It doesn't look like it passes any props to this callback. So you should be able to type this like:

```js
render: () => React.Element<any>,
```

Reply:
Changed it to 
```js 
render: () => React.Node,
 ```